src/bonsai/plots.ml

```diff
@@ -473,7 +503,40 @@ let component (t : t) graph =
         in
         Node.div
           ~attrs:[ Style.plots_container ]
-          [ Node.div ~attrs:[ Style.outcome_chip_wrapper ] [ render_outcome_chip event ]
+          [ Node.div
+              ~attrs:
+                [ {%css|
+                    display: flex;
+                    flex-direction: column;
+                    align-items: center;
+                  |}
+                ]
+              [ Node.div
+                  ~attrs:[ Style.outcome_chip_wrapper ]
+                  [ render_outcome_chip event ]
+              ; (match Event.outcome event with
+                 | Pending -> Node.div []
+                 | Yes explanation | No explanation ->
+                   Node.div
+                     ~attrs:
```

Comment:
Looking at the website locally, I don't enjoy how the All view renders. Could we display the detail only in the Single view?